Sources/RealDeviceMapLib/Controller/DBController.swift

```diff
@@ -255,6 +258,11 @@ public class DBController {
     }
 
     private func backup(mysql: MySQL, fromVersion: Int, toVersion: Int) {
+        if ConfigLoader.global.getConfig(type: .dbBackup) {
```

Comment:
Does this work without `as Bool`?
I tried in #405 but tester reported that if branch is not visited properly

Reply:
I'll check but I think I had issues getting that to work correctly.

Reply:
> I'll check but I think I had issues getting that to work correctly.

yeah maybe you have to add `as Bool`, because it needs the generic type for config value fetching :)